src/shared/Notifications/Notifications.scss

```diff
@@ -1,12 +1,19 @@
 @import '../../scss/global_styles';
 
+
+
 .notifications {
+  &__wrapper {
+    position: fixed;
+    top: 95px;
+    right: 20px;
+    z-index: 999;
```

Comment:
I believe`z-index` might be slightly lower ;) I suggest 2 :)

Reply:
done)